### ISSUE TITLE
Make CustomJSONEncoder a documented public API

### ISSUE THREAD

**Issue by simonw** [simonw/datasette#1983]
It's used by `datasette-geojson` here: https://github.com/eyeseast/datasette-geojson/commit/902bf135a5a33a0dc8264673d00a59a67cb05152

**Comment by simonw** [comment]
https://github.com/simonw/datasette/blob/7b48664d751e59222c01c97a2739d53d6363bdda/datasette/utils/__init__.py#L163-L178

**Comment by eyeseast** [comment]
Yes please, and thank you. I realized I was maybe getting myself in trouble using that, but I think it's a good way to standardize JSON handling.
